docs/siem/detections/api/lists/api-export-list-item.asciidoc

```diff
@@ -0,0 +1,34 @@
+[[lists-api-export-items]]
+=== Export list items
+
+Exports list item values from the specified list container. 
+
+==== Request URL
+
+`POST <kibana host>:<port>/api/lists/items/_export`
+
+
+===== URL query parameters
+
+The URL query must include the list container's `id`:
+
+`id` - `GET /api/lists/items/_export?list_id=<id>`
```

Comment:
Is this right? Did you want `POST /api/lists/items` like you have below and suggest in the docs? It looks like it should be a `POST` here and not a `GET`